src/metabase/lib/drill_thru.cljc

```diff
@@ -3,20 +3,28 @@
    [metabase.lib.drill-thru.column-filter :as lib.drill-thru.column-filter]
    [metabase.lib.drill-thru.common :as lib.drill-thru.common]
    [metabase.lib.drill-thru.distribution :as lib.drill-thru.distribution]
+   [metabase.lib.drill-thru.fk-details :as lib.drill-thru.fk-details]
    [metabase.lib.drill-thru.foreign-key :as lib.drill-thru.foreign-key]
    [metabase.lib.drill-thru.object-details :as lib.drill-thru.object-details]
    [metabase.lib.drill-thru.pivot :as lib.drill-thru.pivot]
+   [metabase.lib.drill-thru.pk :as lib.drill-thru.pk]
    [metabase.lib.drill-thru.quick-filter :as lib.drill-thru.quick-filter]
    [metabase.lib.drill-thru.sort :as lib.drill-thru.sort]
    [metabase.lib.drill-thru.summarize-column :as lib.drill-thru.summarize-column]
    [metabase.lib.drill-thru.summarize-column-by-time :as lib.drill-thru.summarize-column-by-time]
    [metabase.lib.drill-thru.underlying-records :as lib.drill-thru.underlying-records]
+   [metabase.lib.drill-thru.zoom :as lib.drill-thru.zoom]
    [metabase.lib.drill-thru.zoom-in-timeseries :as lib.drill-thru.zoom-in-timeseries]
    [metabase.lib.metadata.calculation :as lib.metadata.calculation]
    [metabase.lib.schema :as lib.schema]
    [metabase.lib.schema.drill-thru :as lib.schema.drill-thru]
    [metabase.util.malli :as mu]))
 
+(comment
+  lib.drill-thru.fk-details/keep-me
+  lib.drill-thru.pk/keep-me
+  lib.drill-thru.zoom/keep-me)
```

Comment:
Moved the methods for these into their own namespaces so it was easier to find them when working on fixing them